demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -0,0 +1,515 @@
+    configure_vscode_tasks(ide_folder, integration_script, test_docker_image)
+    configure_vscode_launch(ide_folder, integration_script)
+    configure_devcontainer(integration_script, test_docker_image)
+
+
+def install_virtualenv(
+    integration_script: IntegrationScript,
+    test_docker_image: str,
+    overwrite_virtualenv: bool,
+) -> Path:
+    """This function installs a virtualenv for the integration script using the test docker image
+
+    Args:
+        integration_script (IntegrationScript): The integration script object
+        test_docker_image (str): The test docker image
+        overwrite_virtualenv (bool): Whether to overwrite existing virtualenv
+
+    Returns:
+        Path: The created virtualenv path
+    """
+    docker_client = docker_helper.init_global_docker_client()
+    requirements = (
+        docker_client.containers.run(
+            test_docker_image, command="pip list --format=freeze", remove=True
+        )
+        .decode()
+        .split("\n")
+    )
+    venv_path = integration_script.path.parent / "venv"
+    interpreter_path = venv_path / "bin" / "python"
+    if venv_path.exists() and not overwrite_virtualenv:
+        logger.info(f"Using existing virtualenv in {venv_path}")
+        return interpreter_path
+    logger.info(f"Creating virtualenv for {integration_script.name}")
+    shutil.rmtree(venv_path, ignore_errors=True)
+    venv.create(venv_path, with_pip=True)
+    for req in requirements:
+        if not req:
+            continue
+        try:
+            subprocess.run(
+                [
+                    f"{venv_path / 'bin' / 'pip'}",
+                    "-q",
+                    "--disable-pip-version-check",
+                    "install",
+                    req,
+                ],
+                check=True,
+            )
+            logger.info(f"Installed {req}")
+        except subprocess.CalledProcessError:
+            logger.warning(f"Could not install {req}, skipping...")
+    return interpreter_path
+
+
+def configure_params(
+    integration_script: IntegrationScript,
+    secret_id: Optional[str],
+    instance_name: Optional[str],
+    test_module: bool,
+) -> None:
+    """Configuring the integration parameters locally and in XSOAR/XSIAM
+
+    Args:
+        integration_script (IntegrationScript): The integration script object configure params to
+        secret_id (Optional[str]): The secret id of the parameters. defaults to the inegration name.
+        instance_name (Optional[str]): The instance name to configure on XSOAR/XSIAM. If None, will not configure.
+        test_module (bool): Whether test-module will run or not.
+    """
+
+    if not secret_id:
+        secret_id = integration_script.name.replace(" ", "_")
+        secret_id = re.sub(r"[()]", "", secret_id)
+    if (project_id := os.getenv("DEMISTO_SDK_GCP_PROJECT_ID")) and isinstance(
+        integration_script, Integration
```

Comment:
what should customers do with `DEMISTO_SDK_GCP_PROJECT_ID`?
lets document this new env var

Reply:
Where should we document?

Reply:
if you have the power to add a README for this command, then i would say to document it there.
I think it would really help

Reply:
added readme